src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,128 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static final String FIRST_VALID_LOGIN = "Trainee";
+    private static final String SECOND_VALID_LOGIN = "Taylor";
+    private static final String THIRD_VALID_LOGIN = "GeorgeTheMagician";
+    private static final String FIRST_VALID_PASSWORD = "password";
+    private static final String SECOND_VALID_PASSWORD = "Swifty";
+    private static final int FIRST_VALID_AGE = 25;
+    private static final int SECOND_VALID_AGE = 18;
+    private static final int THIRD_VALID_AGE = 19;
+    private static final String NON_VALID_LOGIN = "Max";
```

Comment:
Bad name. Why it is non valid?

Reply:
Because login should be at least 6 characters, that's why it non-valid

Reply:
So maybe you should name strings and users that way?
If I see problem with NON_VALID_LOGIN, it's hard to understand, what's going. But if I see problem with THREE_CHARS_LOGIN, I understand much more

Reply:
It make sense, agree with you